zeitschrift-fur-medienwissenschaft.csl

```diff
@@ -20,493 +14,297 @@
     <category field="humanities"/>
     <issn>1869-1722</issn>
     <summary>Style for zfm. Zeitschrift für Medienwissenschaft. Citations in notes; no bibliography.</summary>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2014-09-13T22:01:31+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <!--
-================================================================================
-   TODO
-================================================================================
--->
-  <!--
-================================================================================
-   TERMS
-================================================================================
--->
+  <!-- HINWEISE
```

Comment:
Maybe it is better to move the "Hinweise" to the `<summary>` but I forgot how it is handled exactly. @rmzelle Any thoughts?